git/02.development-workflow.md

```diff
@@ -18,6 +18,10 @@ supercede general best practices.
 Start by creating a branch from the [release branch](07.release-branching.md)
 and switching to it.
 
+```
+$ git checkout -b feature_branch_name
+```
```

Comment:
@calebwoods A lot of these steps could contain example instructions, but I opted not to include them to try to keep things at a high level. Do you think it's important to include a command for this, or would you be alright striking it?